Avoid deprecation warning from Linux 5.6+ build system.

Index: dahdi-linux/drivers/dahdi/Kbuild
===================================================================
--- dahdi-linux.orig/drivers/dahdi/Kbuild	2023-09-05 18:13:48.979887272 +0530
+++ dahdi-linux/drivers/dahdi/Kbuild	2023-09-05 18:14:24.739886778 +0530
@@ -164,6 +164,7 @@
 
 
 hostprogs-y := makefw
+hostprogs := makefw
 
 
 
Index: dahdi-linux/drivers/dahdi/xpp/Kbuild
===================================================================
--- dahdi-linux.orig/drivers/dahdi/xpp/Kbuild	2023-09-05 18:13:48.979887272 +0530
+++ dahdi-linux/drivers/dahdi/xpp/Kbuild	2023-09-05 18:14:48.811886445 +0530
@@ -58,6 +58,7 @@
 FXO_VERIFY = $(obj)/init_card_2_$(XPP_PROTOCOL_VERSION) -v $(obj)/init_fxo_modes
 
 hostprogs-y			:= print_fxo_modes
+hostprogs			:= print_fxo_modes
 always				:= $(xpp_verified)
 print_fxo_modes-objs		:= print_fxo_modes.o
 HOSTCFLAGS_print_fxo_modes.o	+= -include $(FXO_MODES)
